src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const totalCost = days * 40;
```

Comment:
what are 40, 50, 20? fix it according to the checklist, 5th point

Reply:
"fix it according to the checklist, 5th point" i did not understand where t look at ? i was looking here 
https://github.com/mate-academy/style-guides/blob/master/javascript.md#23-naming-conventions

is this right place ? but at 5th point is 5. Destructuring ?

Reply:
https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md

Reply:
oh man feel so stupid now, thank you!  yes i see the file now 😂😂